### PR TITLE
TOC layout

### LINKED ISSUE #144: Sidebar TOC design

### PR DESCRIPTION
**Pull request by janno42**
fix #144, change position of toc in sidebar and handle long links

### CODE REVIEW

File: _1327/information_pages/templates/information_pages_base.html [hunk 1]
```diff
@@ -10,24 +10,28 @@
 {% endblock %}
 
 {% block sidebar %}
-	{{ toc|safe }}
 	<dl class="meta-information">
 		<dt>{% trans "Author" %}</dt>
 		<dd>{{ document.author }}</dd>
 		<dt>{% trans "Created" %}</dt>
-		<dd>{{ document.created }}</dd>
+		<dd>{{ document.created|date:"d. M Y, H:i" }}</dd>
```
Comment (invliD) [inline _1327/information_pages/templates/information_pages_base.html]: Add localization support to the date format.

File: _1327/information_pages/templates/information_pages_base.html [hunk 2]
```diff
@@ -10,24 +10,28 @@
...
 			<a class="btn btn-sm btn-sidebar {% if active_page == 'view' %}btn-warning{% else %}btn-default{% endif %}" href="{% url 'information_pages:view_information' document.url_title %}" role="tab">{% trans "View" %}</a>
 			<a class="btn btn-sm btn-sidebar {% if active_page == 'edit' %}btn-warning{% else %}btn-default{% endif %}" href="{% url 'information_pages:edit' document.url_title %}" role="tab">{% trans "Edit" %}</a>
 			<a class="btn btn-sm btn-sidebar {% if active_page == 'attachments' %}btn-warning{% else %}btn-default{% endif %}" href="{% url 'information_pages:attachments' document.url_title %}" role="tab">{% trans "Attachments" %}</a>
 			<a class="btn btn-sm btn-sidebar {% if active_page == 'permissions' %}btn-warning{% else %}btn-default{% endif %}" href="{% url 'information_pages:permissions' document.url_title %}" role="tab">{% trans "Permissions" %}</a>
 			<a class="btn btn-sm btn-sidebar {% if active_page == 'versions' %}btn-warning{% else %}btn-default{% endif %}" href="{% url 'information_pages:versions' document.url_title %}" role="tab">{% trans "Versions" %}</a>
+			<div class="divider"></div>
 		{% endif %}
 	{% endblock %}
+	<div class="toc">
```
Comment (invliD) [inline _1327/information_pages/templates/information_pages_base.html]: ``{{ toc|safe }}`` already includes a ``div#toc``. You might want to rename this one.

File: _1327/information_pages/templates/information_pages_base.html [hunk 3]
```diff
@@ -46,3 +50,16 @@
...
+
+
+{% block scripts %}
+    {{ block.super }}
+    <script type="text/javascript">
+        $(document).ready(function() {
+        	// add link text to title of all elements in toc to enable tooltips with full text
+        	$('.toc a').each(function() {
+				this.title = this.text;
+			});
```
Comment (invliD) [inline _1327/information_pages/templates/information_pages_base.html]: Indentation is broken here.

**Comment by janno42** [comment]
updated

File: _1327/information_pages/templates/information_pages_base.html (hunk 1 above)
Comment (janno42) [inline _1327/information_pages/templates/information_pages_base.html]: shouldn't this be localized already?

Comment (invliD) [inline _1327/information_pages/templates/information_pages_base.html]: I mean you're explicitly setting the date format. But ``d. M Y`` is not used everywhere.

Comment (janno42) [inline _1327/information_pages/templates/information_pages_base.html]: would be a great time to only use iso-standard format! (yyyy-mm-dd)

Comment (invliD) [inline _1327/information_pages/templates/information_pages_base.html]: As a display date? I don't think so.

Comment (janno42) [inline _1327/information_pages/templates/information_pages_base.html]: what a pity.

Comment (janno42) [inline _1327/information_pages/templates/information_pages_base.html]: won't fix. someone else can do it.

Comment (invliD) [inline _1327/information_pages/templates/information_pages_base.html]: Alright, please create an issue.

**Comment by invliD** [comment]
:+1:

**Pull request merged**